contexts/credentials/v2

```diff
@@ -123,9 +123,9 @@
       }
     },
 
-    "VerifiableCredentialSchema2023": {
+    "JsonSchemaCredential": {
       "@id":
-        "https://w3.org/2018/credentials#VerifiableCredentialSchema2023",
+        "https://www.w3.org/TR/vc-json-schema/#JsonSchemaCredential",
```

Comment:
```suggestion
        "https://w3.org/2018/credentials#JsonSchemaCredential",
```

Fixed to align with this:

https://github.com/w3c/vc-data-model/pull/1218/files